example/rds-cluster-tf/variables.tf

```diff
@@ -48,6 +48,16 @@ variable "cluster_identifier" {
   default     = "cumulus-rds-serverless-default-cluster"
 }
 
+variable "cluster_instance_count" {
+  description = "Number of instances to create inside of the cluster"
+  type = number
+  default = 1
+  validation {
+    condition = var.cluster_instance_count >= 1 && var.cluster_instance_count <= 16
+    error_message = "Variable cluster_instance_count should be between 1 and 16."
+  }
+}
```

Comment:
This is because we're `provisioned` now? Same question as below. Context around this change would be great. I may have missed a conversation or two.

Reply:
This looks to be because of the read replica configuration (work that was scheduled for CUMULUS-3593) - this creates a ACU*instances setup in the cluster. 

Splitting up documentation from development may have been a mistake, we probably need to differentiate between module/readme docs and user docs, but that's not an issue with this PR so much as scope.

Reply:
With Serverless v2, the minimum configuration is a cluster with one instance. The v2 terraform module has a "count" meta-parameter to make multiple instances and join them all to the same RDS cluster. Ref: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rds_cluster_instance

With my template updates in this PR, I went ahead and exposed this count parameter as a terraform input variable, since we know we will need that in a future ticket to allow the creation and utilization of read replicas. The default value of that variable is 1. Ref: https://bugs.earthdata.nasa.gov/browse/CUMULUS-3593

Reply:
Ok, that makes sense. Again here I'm just missing context and need to catch up so this is helpful.